app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,21 @@
+%nav.navbar.navbar-default
+  .container-fluid
+    .navbar-header
+
+      %button.navbar-toggle.collapsed{ type: "button", "data-toggle" => "collapse", "data-target" => "#bs-example-navbar-collapse-1" }
```

Comment:
Maybe change `#bs-example-navbar-collapse-1` to something like `#main-navbar` so it's easier to read.